.codeclimate.yml

```diff
@@ -14,6 +14,7 @@ plugins:
     enabled: true
   rubocop:
     enabled: true
+    channel: rubocop-1-31-0
```

Comment:
![image](https://user-images.githubusercontent.com/25785789/202269772-acbcfde7-a1c4-4743-b40c-a5f8534b918e.png)
From https://docs.codeclimate.com/docs/rubocop

Do you plan on adding one?

Reply:
We already have a `.rubocop.yml` that's compatible with that version

Reply:
Okay cool just checking